scaladoc-testcases/src/tests/extensionParams.scala

```diff
@@ -1,22 +1,54 @@
 package tests.extensionParams
 
+trait Animal
+
 extension [A](thiz: A)
-  def toTuple2[B](that: B): (A, B) = thiz -> that
+  def toTuple2[B](that: B): (A, B)
+   = thiz -> that
 
 extension [A](a: A)(using Int)
-  def f[B](b: B): (A, B) = ???
+  def f1[B](b: B): (A, B)
+  = ???
 
 extension [A](a: A)(using Int)
-  def ff(b: A): (A, A) = ???
+  def f2(b: A): (A, A)
+  = ???
 
 extension [A](a: A)(using Int)
-  def fff(using String)(b: A): (A, A) = ???
+  def f3(using String)(b: A): (A, A)
+  = ???
 
 extension (a: Char)(using Int)
-  def ffff(using String)(b: Int): Unit = ???
+  def f4(using String)(b: Int): Unit
+  = ???
 
 extension (a: Char)(using Int)
-  def fffff[B](using String)(b: B): Unit = ???
+  def f5[B](using String)(b: B): Unit
+  = ???
+
+extension [A <: List[Char]](a: Int)(using Int)
+  def f6[B](b: B): (A, B)
+  = ???
+
+extension [A <: List[Char]](using String)(using Unit)(a: A)(using Int)(using Number)
+  def f7[B, C](b: B)(c: C): (A, B)
+  = ???
+
+extension [A <: List[Char]](using String)(using Unit)(a: A)(using Int)(using Number)
+  def f8(b: Any)(c: Any): Any
+  = ???
+
+extension [A <: List[Char]](using String)(using Unit)(a: A)(using Int)(using Number)
+  def f9[B, C](using Int)(b: B)(c: C): (A, B)
+  = ???
+
+extension [A <: List[Char]](using String)(using Unit)(a: A)(using Int)(using Number)
+  def f10(using Int)(b: Any)(c: Any): Any
+   = ???
+
+  def f12(using Int)(b: A)(c: String): Number
+   = ???
 
-extension [A <: List[Char]](a: A)(using Int)
-  def ffffff[B](b: B): (A, B) = ???
+extension (using String)(using Unit)(a: Animal)(using Int)(using Number)
+  def f11(b: Any)(c: Any): Any
+  = ???
```

Comment:
How do these tests work? I see test “fixtures” but no test specification.

Reply:
I had answered in person, but I'll put it here for posterity:
This works by comparing the text of the file with the output of scala-meta, similarly to what is done in the compiler with the `tests/pos`, `tests/neg` folders.
This is therefore both the declaration and the specification for the tests !